### ISSUE TITLE
Filter scheme properties based on XenonPropertyDescription.Component.SCHEDULER or XenonPropertyDescription.Component.FILESYSTEM

### ISSUE THREAD

**Issue by sverhoeven** [NLeSC/xenon-cli#12]
```
xenon ssh --help
usage: xenon ssh [-h] --location LOCATION [--prop KEY=VALUE] {exec} ...

The SSH adaptor implements all functionality with remote ssh servers.

optional arguments:
  -h, --help             show this help message and exit
  --location LOCATION    Location, Supported locations:
                         - [user@]host[:port]
  --prop KEY=VALUE       Xenon adaptor properties, Supported properties:
                         - xenon.adaptors.ssh.autoAddHostKey=true (Automatically add unknown host keys to known_hosts., type:BOOLEAN) 
                         - xenon.adaptors.ssh.strictHostKeyChecking=true (Enable strict host key checking., type:BOOLEAN) 
                         - xenon.adaptors.ssh.loadKnownHosts=true (Load the standard known_hosts file., type:BOOLEAN) 
                         - xenon.adaptors.ssh.loadSshConfig=true (Load the OpenSSH config file., type:BOOLEAN) 
                         - xenon.adaptors.ssh.sshConfigFile=null (OpenSSH config filename., type:STRING) 
                         - xenon.adaptors.ssh.agent=false (Use a (local) ssh-agent., type:BOOLEAN) 
                         - xenon.adaptors.ssh.agentForwarding=false (Use ssh-agent forwarding, type:BOOLEAN) 
                         - xenon.adaptors.ssh.queue.pollingDelay=1000 (The polling delay for monitoring running jobs (in milliseconds)., type:LONG) 
                         - xenon.adaptors.ssh.queue.multi.maxConcurrentJobs=4 (The maximum number of concurrent jobs in the multiq.., type:INTEGER) 
                         - xenon.adaptors.ssh.gateway=null (The gateway machine used to create an SSH tunnel to the target., type:STRING) 

commands:
  {exec}
    exec                 Execute job at location

```
The properties shown here are for newXenon and newScheduler they should be filtered.

**Comment by sverhoeven** [comment]
Blocked by https://github.com/NLeSC/Xenon/issues/430

**Comment by sverhoeven** [comment]
Fixed in 2f2bd52c6b23710a7d6ff19a79bbad9fbf9265e1
